src/pages/SetPasswordPage.js

```diff
@@ -117,29 +121,24 @@ class SetPasswordPage extends Component {
                     shouldShowWelcomeText
                     welcomeText={this.props.translate('setPasswordPage.passwordFormTitle')}
                 >
-                    {_.isEmpty(error) ? (
-                        <>
-                            <View style={[styles.mb4]}>
-                                <NewPasswordForm
-                                    password={this.state.password}
-                                    updatePassword={password => this.setState({password})}
-                                    updateIsFormValid={isValid => this.setState({isFormValid: isValid})}
-                                    onSubmitEditing={this.validateAndSubmitForm}
-                                />
-                            </View>
-                            <View>
-                                <Button
-                                    success
-                                    style={[styles.mb2]}
-                                    text={buttonText}
-                                    isLoading={this.props.account.loading || this.props.userSignUp.isValidatingEmail}
-                                    onPress={this.validateAndSubmitForm}
-                                    isDisabled={!this.state.isFormValid}
-                                />
-                            </View>
-                        </>
-                    )
-                        : (<Text>{error}</Text>)}
+                    <View style={[styles.mb4]}>
+                        <NewPasswordForm
+                            password={this.state.password}
+                            updatePassword={password => this.setState({password})}
+                            updateIsFormValid={isValid => this.setState({isFormValid: isValid})}
+                            onSubmitEditing={this.validateAndSubmitForm}
+                        />
+                    </View>
+                    <View>
+                        <FormAlertWithSubmitButton
+                            buttonText={buttonText}
```

Comment:
NAB: the console log is throwing a warning message saying that `onFixTheErrorsLinkPressed` property is required. I think we could add something like `onFixTheErrorsLinkPressed={() => {}}` in `FormAlertWithSubmitButton` :) 

<img width="745" alt="Screen Shot 2022-04-19 at 11 23 51" src="https://user-images.githubusercontent.com/6829422/164050495-8c3b7a62-7098-437c-b193-198b6da53fdc.png">

Reply:
Nice catch! It's a blocker for me so I'll fix it.